pkg/azurefile/azurefile.go

```diff
@@ -755,15 +755,15 @@ func (d *Driver) CreateFileShare(accountOptions *azure.AccountOptions, shareOpti
 }
 
 // DeleteFileShare deletes a file share using storage account name and key
-func (d *Driver) DeleteFileShare(subsID, resourceGroup, accountName, shareName string, secrets map[string]string) error {
+func (d *Driver) DeleteFileShare(ctx context.Context, subsID, resourceGroup, accountName, shareName string, secrets map[string]string) error {
 	return wait.ExponentialBackoff(d.cloud.RequestBackoff(), func() (bool, error) {
 		var err error
 		if len(secrets) > 0 {
 			accountName, accountKey, rerr := getStorageAccount(secrets)
 			if rerr != nil {
 				return true, rerr
 			}
-			err = d.fileClient.deleteFileShare(accountName, accountKey, shareName)
+			err = d.fileClient.deleteFileShare(ctx, accountName, accountKey, shareName)
```

Comment:
I think original issue is the test is running on L768: `d.cloud.DeleteFileShare(subsID, resourceGroup, accountName, shareName)` it's stuck there.

Reply:
thanks, let's merge this PR first.